.github/workflows/alert-sync-issues.yml

```
@@ -0,0 +1,16 @@
+    - cron: '17 * * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  sync:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Alert sync skipped - phenoShared deprecated
+        run: echo "Alert sync workflow disabled due to phenoShared deprecation. TODO: Implement custom alert sync if needed."
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)